esmvalcore/_recipe.py

```diff
@@ -365,71 +364,85 @@ def _get_default_settings(variable, config_user, derive=False):
 def _add_fxvar_keys(fx_var_dict, variable):
     """Add keys specific to fx variable to use get_input_filelist."""
     fx_variable = dict(variable)
+    fx_variable.update(fx_var_dict)
 
     # set variable names
     fx_variable['variable_group'] = fx_var_dict['short_name']
-    fx_variable['short_name'] = fx_var_dict['short_name']
 
     # specificities of project
     if fx_variable['project'] == 'CMIP5':
-        fx_variable['mip'] = 'fx'
         fx_variable['ensemble'] = 'r0i0p0'
-    elif fx_variable['project'] == 'CMIP6':
-        fx_variable['grid'] = variable['grid']
-        if 'mip' in fx_var_dict:
-            fx_variable['mip'] = fx_var_dict['mip']
     elif fx_variable['project'] in ['OBS', 'OBS6', 'obs4mips']:
         fx_variable['mip'] = 'fx'
```

Comment:
I don't think this is necessary, since the `mip` is dictated by the function calling `_add_fxvar_keys`

Reply:
good call, just removed it :beer: